package.json

```diff
@@ -7,18 +7,26 @@
   "license": "UPL",
   "private": true,
   "scripts": {
-    "build": "webpack --config webpack.prod.js && npm run test",
+    "build": "npm-run-all -l build:notests test",
     "build:notests": "webpack --config webpack.prod.js",
     "clean": "rimraf dist",
     "start:dev": "webpack serve --hot --color --progress --config webpack.dev.js",
     "test": "jest --maxWorkers=50% --coverage=true",
     "test:ci": "jest --maxWorkers=50%",
-    "eslint": "eslint --ext .tsx,.js ./src/",
+    "eslint": "npm-run-all eslint:check",
+    "eslint:check": "eslint --ext .ts,.tsx,.js ./src/",
+    "eslint:apply": "eslint --ext .ts,.tsx,.js --fix ./src/",
     "license:check": "license-check-and-add check -f license-config.json",
-    "lint": "npm run license-check-and-add && npm run eslint",
-    "format": "prettier --check --write ./src/**/*.{tsx,ts} && license-check-and-add add -f license-config.json",
+    "license:apply": "license-check-and-add add -f license-config.json",
+    "lint": "npm-run-all -l -p format:check eslint",
+    "format": "npm-run-all format:apply",
```

Comment:
Here `format` is alias for `format:apply` but above, `eslint` is alias for `eslint:check`? Should we make it consistent?